templates/status.html

```diff
@@ -14,63 +14,85 @@
+    <p>The status of each backend which can run jobs for OpenSAFELY researchers can be viewed below.</p>
+
+    <h3>TPP</h3>
+    {% db_maintenance_banner %}
+
+    <div class="grid gap-8 mb-8">
+      {% for backend in backends %}
+        {% #card %}
+          <div class="px-4 py-2 sm:px-6 sm:py-3">
+            <div class="flex flex-wrap items-center justify-between sm:flex-nowrap">
+              <h2 id="workspaces-title" class="text-xl font-semibold tracking-tight text-slate-900">
+                {{ backend.name }}
+              </h2>
+              {% if backend.show_warning %}
+                {% pill variant="warning" text="Missing" %}
+              {% endif %}
+            </div>
           </div>
-        </div>
+          <div>
+            {% #description_item title="Running jobs" stacked=True %}
+              {{ backend.queue.running }}
+            {% /description_item %}
+            {% #description_item title="Pending jobs" stacked=True %}
+              {{ backend.queue.pending }}
+            {% /description_item %}
+            {% #description_item title="Last contact with "|add:backend.name|add:" backend" stacked=True %}
+              <time
+                class="
+                       text-slate-900 group relative cursor-pointer
+                       {% if not backend.last_seen %}text-bn-ribbon-900{% endif %}
+                      "
+                datetime="{{ backend.last_seen|date:"Y-m-d H:i:sO" }}">
+                {{ backend.last_seen|naturaltime }}
+                {% if backend.last_seen %}
+                  {% tooltip class="font-mono" position="-bottom-4" content=backend.last_seen|date:"d M Y H:i:s e" %}
+                {% endif %}
+              </time>
+            {% /description_item %}
+            {% #description_item title="Job requests not yet processed by "|add:backend.name|add:" backend" stacked=True %}
+              {{ backend.queue.unacked|floatformat:"-3g" }}
+            {% /description_item %}
+            {% #description_item title="Total job requests received by "|add:backend.name|add:" backend" stacked=True %}
+              {{ backend.queue.acked|floatformat:"-3g" }}
+            {% /description_item %}
+          </div>
+          {% #card_footer no_container class="border-t border-t-slate-200" %}
+            <p class="text-sm font-medium text-slate-600">
+              {{ backend.name }} backend is missing if OpenSAFELY Jobs does
+              not receive a response for {{ backend.alert_timeout|duration }}.
+            </p>
+          {% /card_footer %}
+        {% /card %}
+      {% endfor %}
+    </div>
+
+
+    <h2 id="db-maintenance-mode">Database maintenance mode</h2>
+    <p>To ensure you're always working with the most up-to-date data, the database undergoes routine maintenance once a week. During this time, the system will be temporarily unavailable while we update the data.
```

Comment:
This is only true for TPP. We don't know what maintenance mode will look like for EMIS or other backends. It's also possible that it may change away from being updated weekly at some point. It might be better to say that the backend databases undergo regular routine maintenance, or something less specific, or we can make a note to remind ourselves to update this when it changes.